pandas/tests/reshape/test_crosstab.py

```diff
@@ -698,3 +698,46 @@ def test_margin_normalize(self):
             names=["A", "B"],
         )
         tm.assert_frame_equal(result, expected)
+
+    def test_crosstab_multiple_columns_normalize(self):
```

Comment:
is it possible to parametrize the tests added in #27663, would the results be the expected transposed?

Reply:
Wouldn't that result in a lot of overhead defining the parameters (inputs and results)?  I actually see no easy way to refactor these usiing `@pytest.mark.parametrize`.

What do you mean by `would the results be the expected transposed?`?